BatteryParamGeneration.test.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from Predefined_values import NumOfItteration, limit
+import GenerateSenderOutput
+
+class RandomGeneratorTest(unittest.TestCase):
+    def test_Itteration_CountType(self):
+        self.assertTrue(GenerateSenderOutput.ConvertItterationCountToString(2) == '2')
+        self.assertTrue(GenerateSenderOutput.ConvertItterationCountToString('2') == '2')
+     
+    def test_NoOfItteration_MorethanMin(self):
+        self.assertTrue(GenerateSenderOutput.RandomGenerateParameter(4) == 'Minimum itteration should be more than 5')
+        self.assertTrue(GenerateSenderOutput.RandomGenerateParameter(7) == 'Battery parameter generated for Receiver')
+        
+    def test_SenderOutput_Correctly_Generated(self):
+        #self.assertTrue(GenerateSenderOutput.SenderOutputOrReceiverInput([[['temperature 0', 76, 'check_limit'], ['state_of_charge 0', 22, 'Limit OK']]]) == True)

```

Comment:
Why is this commented? A maintainer would find this nervous... remove commented code in commits.